lints/lint_old_root_ca_rsa_mod_less_than_2048_bits.go

```diff
@@ -21,14 +21,12 @@ func (l *rootCaModSize) Initialize() error {
 
 func (l *rootCaModSize) CheckApplies(c *x509.Certificate) bool {
 	issueDate := c.NotBefore
-	return (c.PublicKeyAlgorithm == x509.RSA && util.IsRootCA(c) && issueDate.Before(util.RsaDate2))
+	_, ok := c.PublicKey.(*rsa.PublicKey)
+	return (c.PublicKeyAlgorithm == x509.RSA && util.IsRootCA(c) && issueDate.Before(util.RsaDate2)) && ok
```

Comment:
@zakird There's a lint that checks whether or not a public key exists already, so rather than returning a fatal when there's no public key, I think it's fine if we just don't say it's relevant for all the other RSA checks rather than returning fatal